README.md

```diff
@@ -32,3 +32,31 @@ found at the following links:
 
 + https://help.github.com/articles/creating-a-pull-request/ 
 + https://help.github.com/articles/autolinked-references-and-urls/
+
+
+## How to set up the API
+1. Before you can access the API, you need to install docker toolbox and docker-for-Mac (or whichever is appropriate for your machine)
+
+    https://www.docker.com/get-docker
+
+2. Once you have those installed, run the following command in your terminal:
+```sh 
+touch .env
```

Comment:
I think https://github.com/charlottejuniordevs/api/issues/13 applies here.  Do you mind incorporating that into this PR as well?